packages/experimental/Badge/src/BadgeTokens.ts

```diff
@@ -70,4 +69,10 @@ export const defaultBadgeTokens: TokenSettings<BadgeTokens, Theme> = () =>
     square: {
       borderRadius: globalTokens.corner.radius.none,
     },
+    rtl: {
+      left: globalTokens.spacing.none,
+    },
+    ltr: {
+      right: globalTokens.spacing.none,
+    },
```

Comment:
Dumb question that you're free to reject: Would it work if we named `right` something like `start` or `startOffset` and then didn't need to set a different token based on rtl or ltr? AKA, move the check of whether to add spacing to the left or right based on RTL out the token and into the styling file?

Reply:
I removed LTR. It worked with `right` prop is undefined when `left` is set